ha/core/cluster/cluster_manager.py

```diff
@@ -65,11 +66,18 @@ def __init__(self):
         """
         super(PcsClusterManager, self).__init__()
         self._execute = SimpleCommand()
-        self._decision_monitor = DecisionMonitor()
-        # TODO: add node_manager class to handle query
-        self._refresh_contex = PcsRefreshContex(self._decision_monitor)
-        # TODO move node logic to node manager class
-        self._node_status = [ 'Online', 'Standby', 'Maintenance', 'Offline', 'Disconnected']
+
+        # get version from ha.conf
+        version = Conf.get(const.HA_GLOBAL_INDEX, "VERSION.version")
```

Comment:
Please add this code in ConfigManager class, inside a function - get_major_version()

Reply:
okay

Reply:
done